src/Cache/Region/FileLockRegion.php

```diff
@@ -158,7 +158,7 @@ public function evictAll(): bool
         // when nothing matched (even though no errors occurred)
         $filenames = glob(sprintf('%s/*.%s', $this->directory, self::LOCK_EXTENSION));
```

Comment:
Shall we eliminate `false` instead? We treat falsy values that `glob()` might return (`false` and `[]`) the same way, so might as well normalize the value. That would allow us to unwrap the `if` block below.

```suggestion
        $filenames = glob(sprintf('%s/*.%s', $this->directory, self::LOCK_EXTENSION)) ?: [];
```

Reply:
Good point, done 👍